includes/admin/payments/orders.php

```diff
@@ -276,42 +298,60 @@ function edd_order_details_email( $order ) {
 
 		$all_emails[ $key ] = $email;
 	}
+
+	$help = __( 'Send a new copy of the purchase receipt to the email address used for this order. If download URLs were included in the original receipt, new ones will be included.', 'easy-digital-downloads' );
 ?>
 
 	<div><?php
 		if ( ! empty( $customer->emails ) && count( (array) $customer->emails ) > 1 ) : ?>
-			<fieldset class="edd-order-resend-email-chooser">
-				<legend>
+			<fieldset class="edd-form-group">
+				<legend class="edd-form-group__label">
 					<?php _e( 'Send email receipt to', 'easy-digital-downloads' ); ?>
 				</legend>
 
 				<?php foreach ( $all_emails as $key => $email ) : ?>
-				<p>
+				<div class="edd-form-group__control is-radio">
+					<input autocomplete="off" id="<?php echo rawurlencode( sanitize_email( $email ) ); ?>" class="edd-form-group__input" name="edd-order-resend-receipt-address" type="radio" value="<?php echo rawurlencode( sanitize_email( $email ) ); ?>" <?php checked( true, ( 'primary' === $key ) ); ?> />
+
 					<label for="<?php echo rawurlencode( sanitize_email( $email ) ); ?>">
-						<input autocomplete="off" id="<?php echo rawurlencode( sanitize_email( $email ) ); ?>" class="edd-order-resend-receipt-email" name="edd-order-resend-receipt-address" type="radio" value="<?php echo rawurlencode( sanitize_email( $email ) ); ?>" <?php checked( true, ( 'primary' === $key ) ); ?> />
 						<?php echo esc_attr( $email ); ?>
 					</label>
-				</p>
+				</div>
 				<?php endforeach; ?>
+
+				<p class="edd-form-group__help">
```

Comment:
@robincornett I think it's fine to add `.description`, but I'd like to keep the styles on `.edd-form-group__help` since core includes `p` or `span` specificity as well as alters it based on additional contexts: https://github.com/WordPress/WordPress/blob/fcff974c33f9bf2419d0d9f22ea7df8004e1122c/wp-admin/css/forms.css#L1495